sdmetrics/reports/multi_table/quality_report.py

```diff
@@ -109,6 +111,43 @@ def get_properties(self):
             'Score': self._property_breakdown.values(),
         })
 
+    def show_details(self, property_name, table_name=None):
+        """Display a visualization for each score for the given property and table.
+
+        Args:
+            property_name (str):
+                The name of the property to return score details for.
+            table_name (str):
+                The table to show scores for. Must be provided for 'Column Shapes'
+                and 'Column Pair Trends'
+        """
+        if property_name in ['Column Shapes', 'Column Pair Trends'] and table_name is None:
+            raise ValueError('Table name must be provided when viewing details for '
+                             f'property {property_name}.')
+
+        if property_name == 'Column Shapes':
+            score_breakdowns = {
+                metric.__name__: self._metric_results[metric.__name__].get(table_name, {})
```

Comment:
minor: maybe we can pull out `metric.__name__` and `self._metric_results[metric.__name__]` into variables outside this if else block since it seems like all the statements use them

Reply:
Oh, that's a little hard to do since `metric` is only defined in the inline for loop.